build.md

```diff
@@ -0,0 +1,68 @@
+# Building your project's front-end
+
+We have a standard. Please stick to it.
+
+Every CFPB project with a non-trivial front-end should:
+
+ 1. Use **Grunt** to accomplish all front-end build tasks.
+ 1. Use **Less** as a CSS pre-processor.
+ 1. Use an optional **config.json** file in the same directory as the gruntfile to store build variables (internal API endpoints, PII, etc.). This file should be consumed by a Grunt task and listed in `.gitignore`.
+ 1. Only require **two commands** to install dependencies and build all static assets:
+   1. `npm install`
+   1. `grunt build`
+ 1. These two commands (and any additional commands if there's a strong argument for them) should be stored in a frontendbuild.sh file to make things easier for our CI environments.
```

Comment:
It makes CI easier. There's only one step to tell Travis and Jenkins. I'm indifferent about the "only require two commands" part but I've found it has encouraged devs to keep their build processes simple so I support leaving it in.